### DISCUSSION TITLE
Lazer Turned on my mouse acceleration in win 10?

### DISCUSSION THREAD

**Discussion by OwOssyUwU** [ppy/osu#17335] [Q&A]
I'm really not sure what logs would help with this, but I'm a relative tablet user who uses opentabletdriver, my tablet is a CTH-470. Relative tablet isn't an option so I have to use mouse settings, I turned off high precision mouse, realized I couldn't adjust sensitivity if I did so, and that my opentabletdriver sensitivity was being ignored, so I turned it back on. Went back to stable after a while and something felt SUPER wrong so I checked my driver, was set up how I left it, but it felt like I had the nastiest acceleration in the world, trying to play a low star map was actually impossible. Checked win 10 and my mouse acceleration was turned on when it's been off since long before I even downloaded osu, and I had been playing stable with no problems immediately before launching lazer. not sure what else to say, but I'm wondering how lazer even has the perms to do that without telling me to be honest.

Edit: went back to lazer to see if turning high precision mouse on and off again was what did it, I'm not certain but it didn't happen again.

**Comment by frenzibyte** [comment]
I'm not sure this discussion thread is actionable if you can't reproduce the issue itself reliably. osu!(lazer) should never be able to configure your mouse settings, and I honestly doubt it has anything to do with SDL at all.
